ts/package-lock.json

```diff
@@ -7,8 +7,8 @@
 			"name": "@microsoft/dev-tunnels",
 			"license": "MIT",
 			"dependencies": {
-				"@microsoft/dev-tunnels-ssh": "^3.11.11",
-				"@microsoft/dev-tunnels-ssh-tcp": "^3.11.11",
+				"@microsoft/dev-tunnels-ssh": "^3.11.13",
+				"@microsoft/dev-tunnels-ssh-tcp": "^3.11.13",
```

Comment:
There are some autogenerated parts of this file as well. Can you run 'npm install' in this folder to regenerate this file